api/api/settings.py

```diff
@@ -75,8 +76,13 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql_psycopg2",
```

Comment:
We should use `django.db.backends.postgresql` as suggested by [the docs](https://docs.djangoproject.com/en/4.2/ref/databases/#postgresql-notes)